app/scripts/views/sign_in_unblock.js

```diff
@@ -54,12 +54,15 @@ define(function (require, exports, module) {
     },
 
     onSignInError (account, password, err) {
+      const unblockCode = this.getElementValue('#unblock_code');
```

Comment:
This should get the element instead of element value:

`const unblockElement = this.$('#unblock_code');`

Reply:
Sure I'll change this